SRM release notes for version 797xx (#2253)
harness/developer-hub: release-notes/service-reliability-management.md

  <TabItem value="Fixed issues">
```

- When configuring a health source, unable to input zero (0) in the **Lesser Than** and **Greater Than** fields. (SRM-14326)
  
  This issue has been fixed.  You can now input zero (0) in both the fields.

> - Unable to select the environment and service at the account, organization, and project levels when using a template (that has service and environment as input values) to create a monitored service in a Verify step. (SRM-14944)

Reviewer: ```suggestion
- When using a template that has service and environment as input values to create a monitored service in a Verify step, users were unable to select the environment and service at the account, organization, or project levels. (SRM-14944)
```

>   This issue has been resolved. Now, you can select the desired environment and service at the account, organization, and project levels when creating a monitored service using a template during pipeline execution.

Reviewer: ```suggestion
  This issue has been resolved, and now you can select the desired environment and service in these cases as expected.
```

Author: Updated

- `SocketTimedOut` error messages were not displayed in the call log. (OIP-537)
  
  This issue has been fixed. Enhanced the call log to show `SocketTimedOut` error messages.

- In the metrics analysis section, time-series graphs exhibit data shifting when the first applicable timestamp is missing. (OIP-526)

>   This was occurring because the graphs relied on the data itself to determine the starting timestamp rather than using explicit start timestamps for control and test data. This issue has been resolved. Now, the graphs reflect the intended starting points by explicitly including the control and test data start timestamps.

Reviewer: ```suggestion
  This issue has been fixed, and now the graphs reflect the intended starting points.
```

Reviewer: You don't need to explain why's/how's of how we fixed it.

Author: Updated.
